src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -305,4 +305,16 @@
       it { expect(flash[:error]).to eq('Incident projects shall only create below maintenance projects.') }
     end
   end
+
+  describe 'GET #linking_projects' do
+    before do
+      login user_moi
+      apache2_project
+      another_project.projects_linked << apache_project
+      xhr :get, :linking_projects, project: apache_project
+    end
+
```

Comment:
And also I see it for API actions that speak HTTP... but in webui actions that don't require login and also don't play with the status itself... I don't see it